8Knot/assets/color.css

```diff
@@ -32,25 +56,32 @@ blockquote p {
   max-width: 950px;
 }
 
-/* this is to change the color for the spinner, cant be changed directly */
+/* ==== DASH SPECIFIC COMPONENTS ==== */
+/* Spinner needs !important because it's deeply nested in Dash internals */
 .dash-spinner.dash-default-spinner {
-  filter: contrast(1.2) hue-rotate(308deg) grayscale(0.76) !important;
+  filter: contrast(1.2) hue-rotate(308deg) grayscale(0.76);
 }
 
+/* Utility classes */
 .explanation-p {
   width: auto;
   min-width: 50px;
   max-width: 550px;
 }
 
-.navbar-brand {
-  color: #6b8976 !important;
+/* Navigation brand with higher specificity */
+.navbar .navbar-brand,
+nav .navbar-brand {
+  color: var(--color-primary);
 }
 
-.metric_data {
-  color: #000000 !important;
-  text-shadow: none !important;
-  padding-bottom: 15px !important;
+/* ==== DATA DISPLAY COMPONENTS ==== */
+/* Metric data with higher specificity */
+.metric-container .metric_data,
+div.metric_data {
```

Comment:
the comma here acts like an "or"

Do you know if both of these selectors are being used? or is it just one of them?

Reply:
@yyualvin